tests/test_fmi.py

```diff
@@ -61,18 +77,99 @@ def solve(self):
 FMU_PATHS.ME1.nominal_test4    = os.path.join(file_path, "files", "FMUs", "XML", "ME1.0", "NominalTest4.fmu")
 FMU_PATHS.ME2.nominal_test4    = os.path.join(file_path, "files", "FMUs", "XML", "ME2.0", "NominalTests.NominalTest4.fmu")
 
+REFERENCE_FMU_PATH = Path(file_path) / 'files' / 'reference_fmus' / '3.0'
+
+# TODO: Many tests here could be parameterized
+# However, in many cases this relies on having FMUs with the same functionality
+# available for different FMI versions.
```

Comment:
I propose we parameterize more tests (as possible) here as the functionality becomes relevant for FMI3.